### DISCUSSION TITLE
whats the purpose of nextcloud.username and nexcloud.password in the helm chart values?

### DISCUSSION THREAD

**Discussion by migs35323** [nextcloud/helm#396] [Q&A]
Started a nexcloud deployment and i filled the values for nextcloud.username and nexcloud.password
when i did open the gui nexcloud is asking me for an admin account which can't be the same that i specified in the values

so whats the purpose of the chart values?

**Comment by provokateurin** [comment]
Those values are the admin account. Maybe the should be named more clearly.

>  when i did open the gui nexcloud is asking me for an admin account which can't be the same that i specified in the values

What do you mean with that? Did you try logging in with the credentials set in the values file?
